src/dawn/Compiler/Options.inc

```diff
@@ -40,6 +40,14 @@ OPT(std::string, Backend, "gridtools", "backend", "",
     "\n - c++-naive   = naive C++ code\n"
     "\n - c++-opt     = optimized C++ code"
     "\n - cuda        = optimized cuda", "<backend>", true, false)
+OPT(std::string, arch, "", "arch", "",
+    "Set the computing architecture. Possible values for cuda backend are:"
+    "\n - K80\n"
+    "\n - P100\n", "<arch>", true, false)
```

Comment:
you have less options in here then you support. Again, I suggest to drop it allthogether